aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -145,7 +145,12 @@ OperatorEntry::AnnotatedKernelContainerIterator OperatorEntry::registerKernel(
 #ifdef C10_DISPATCHER_ONE_KERNEL_PER_DISPATCH_KEY
   if (k[0].kernel.isValid()) {
 #else
-  if (k.size() > 0) {
+  if (k.size() > 0 &&
+        // Don't complain if a PythonOpRegistrationTrampoline is overriding
+        // another PythonOpRegistrationTrampoline; you will have a
+        // registration per multipy/torchdeploy interpreter
+        !(k.front().kernel.isPythonOpRegistrationTrampoline() &&
```

Comment:
is this to let multiple interpreters register a kernel for the same op, dispatch key pair?

Reply:
Yes. I thought about a few ways of doing this, but the easiest was to just let the interpreters clobber each other without raising warnings.